src/main.lua

```diff
@@ -544,6 +573,9 @@ function updateGame()
 		end -- timer is over timerLimit
 	else
 		refreshNeeded = true
+		if showGameOver then
+			drawGameOver()
+		end
```

Comment:
well it just seems like you shoehorned the game over drawing in the code for ending the game
I could leave it like this... OR I could modify the theme engine to also show a game over popup thing...